formspree/templates/forms/unconfirm.html

```diff
@@ -0,0 +1,67 @@
+{% extends 'layouts/message.html' %}
+
+{% block base %}
+<div class="container narrow card">
+  <div class="container" id="header">
+    <div class="col-1-1 header info">
+      <i class="ion ion-help-circled"></i>
+    </div>
+  </div>
+
+  <div class="container">
+    <div class="col-1-1">
+      <h2>You'll not receive submissions for the form on {{ disabled_form.host }} targeting {{ disabled_form.email }} anymore.</h1>
+    </div>
+  </div>
+
+  {% if other_forms.count() %}
+  <div class="container">
+    <div class="col-1-1">
+      <h3>Do you want to unconfirm these other forms too?</h3>
+      <form
+        action="{{ url_for('unconfirm_multiple') }}"
+        method="POST"
+      >
+        <div style="text-align: left; padding-left: 5%">
+          {% if other_forms.count() > 1 %}
+            <label> <input type="checkbox" id="disable-all">Select all</label>
+            <hr>
+          {% endif %}
+          {% for f in other_forms %}
+            <div class="float-left"><label>
+              <input type="checkbox" name="form_ids" value="{{ f.id }}"> {{ f.host }}
+            </label></div>
+          {% endfor %}
+        </div>
+
+        <br>
+        <button disabled type="submit">Yes, I want to stop receiving submissions from these other forms</button>
+      </form>
+    </div>
+  </div>
+  <script>
+    var checkAll = document.getElementById('disable-all')
+    var all = document.querySelectorAll('input[type="checkbox"]')
+    checkAll.addEventListener('change', function () {
```

Comment:
This will not run if there is only 1 other unconfirmed form bc checkAll will return null, causing the whole block to stop executing. Perhaps we should simplify things and make the select all option available as long as there's 1 other form.

Reply:
I think I've fixed it with an `if (checkAll)`.